contracts/bprotocol/avatar/AbsCToken.sol

```diff
@@ -211,6 +211,12 @@ contract AbsCToken is Cushion {
         return cToken.approve(spenderAvatar, amount);
     }
 
+    function collectCToken(ICToken cToken, address from, uint256 cTokenAmt) public {
```

Comment:
should do `postPoolOp`.
Should do `postPoolOp` for dest also in `transfer` and `transferFrom`

Reply:
fixed `postPoolOp` in `collectCToken`